[ao][docs] Clarify operator support for quantization

raghuramank100 opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* __->__ #63270
* #63582

Summary:
Add table to quantization main page showing supported modules
for static and dynamic quantization.

Test Plan:

Reviewers:

Subscribers:

Tasks:

Tags:

Differential Revision: [D30658654](https://our.internmc.facebook.com/intern/diff/D30658654)

raghuramank10000 committed:
[ao][docs] Clarify operator support for quantization
Summary:
Add table to quantization main page showing supported modules
for static and dynamic quantization.

Test Plan:

Reviewers:

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

raghuramank10000 committed:
Update on "[ao][docs] Clarify operator support for quantization"


Summary:
Add table to quantization main page showing supported modules
for static and dynamic quantization.

Test Plan:

Reviewers:

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

raghuramank10000 committed:
Update on "[ao][docs] Clarify operator support for quantization"


Summary:
Add table to quantization main page showing supported modules
for static and dynamic quantization.

Test Plan:

Reviewers:

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

raghuramank10000 committed:
Update on "[ao][docs] Clarify operator support for quantization"


Summary:
Add table to quantization main page showing supported modules
for static and dynamic quantization.

Test Plan:

Reviewers:

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

raghuramank10000 committed:
Update on "[ao][docs] Clarify operator support for quantization"


Summary:
Add table to quantization main page showing supported modules
for static and dynamic quantization.

Test Plan:

Reviewers:

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

raghuramank10000 committed:
Update on "[ao][docs] Clarify operator support for quantization"


Summary:
Add table to quantization main page showing supported modules
for static and dynamic quantization.

Test Plan:

Reviewers:

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

raghuramank10000 committed:
Update on "[ao][docs] Clarify operator support for quantization"


Summary:
Add table to quantization main page showing supported modules
for static and dynamic quantization.

Test Plan:

Reviewers:

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

vkuzo commented on docs/source/quantization.rst:
```diff
@@ -147,23 +147,54 @@ The following table compares the differences between Eager Mode Quantization and
+|nn.Conv1d/2d/3d  | Y                 | N                 |
++-----------------+-------------------+-------------------+
+|nn.LSTM          | N                 | Y                 |
+|nn.GRU           |                   |                   |
```
nit: specify for each line, to stay consistent with L170:L171?

vkuzo commented on docs/source/quantization.rst:
```diff
@@ -147,23 +147,54 @@ The following table compares the differences between Eager Mode Quantization and
+|nn.GRU           |                   |                   |
++-----------------+-------------------+-------------------+
+|nn.RNNCell       | N                 | Y                 |
+|nn.GRUCell       |                   |                   |
+|nn.LSTMCell      |                   |                   |
```
same

vkuzo commented on docs/source/quantization.rst:
```diff
@@ -147,23 +147,54 @@ The following table compares the differences between Eager Mode Quantization and
++-----------------+-------------------+-------------------+
+|nn.Embedding     | Y                 | N                 |
++-----------------+-------------------+-------------------+
+|nn.MHA           |* Supported via    |                   |
```
nit: can we spell out the name (not everyone will know what MHA means), and should we consider not saying this is supported yet until there is a clear long term strategy and clear documentation?

raghuramank100 replied:
Agreed, will change that.

raghuramank10000 committed:
Update on "[ao][docs] Clarify operator support for quantization"


Summary:
Add table to quantization main page showing supported modules
for static and dynamic quantization.

Test Plan:

Reviewers:

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

raghuramank10000 committed:
Update on "[ao][docs] Clarify operator support for quantization"


Summary:
Add table to quantization main page showing supported modules
for static and dynamic quantization.

Test Plan:

Reviewers:

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

raghuramank10000 committed:
Update on "[ao][docs] Clarify operator support for quantization"


Summary:
Add table to quantization main page showing supported modules
for static and dynamic quantization.

Test Plan:

Reviewers:

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

raghuramank10000 committed:
Update on "[ao][docs] Clarify operator support for quantization"


Summary:
Add table to quantization main page showing supported modules
for static and dynamic quantization.

Test Plan:

Reviewers:

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

raghuramank10000 committed:
Update on "[ao][docs] Clarify operator support for quantization"


Summary:
Add table to quantization main page showing supported modules
for static and dynamic quantization.

Test Plan:

Reviewers:

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

raghuramank10000 committed:
Update on "[ao][docs] Clarify operator support for quantization"


Summary:
Add table to quantization main page showing supported modules
for static and dynamic quantization.

Test Plan:

Reviewers:

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

raghuramank100 commented:
@raghuramank100 has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D30658654).

raghuramank10000 committed:
Update on "[ao][docs] Clarify operator support for quantization"


Summary:
Add table to quantization main page showing supported modules
for static and dynamic quantization.

Test Plan:

Reviewers:

Subscribers:

Tasks:

Tags:

Differential Revision: [D30658654](https://our.internmc.facebook.com/intern/diff/D30658654)

[ghstack-poisoned]

raghuramank10000 committed:
Update on "[ao][docs] Clarify operator support for quantization"


Summary:
Add table to quantization main page showing supported modules
for static and dynamic quantization.

Test Plan:

Reviewers:

Subscribers:

Tasks:

Tags:

Differential Revision: [D30658654](https://our.internmc.facebook.com/intern/diff/D30658654)

[ghstack-poisoned]

raghuramank10000 committed:
Update on "[ao][docs] Clarify operator support for quantization"


Summary:
Add table to quantization main page showing supported modules
for static and dynamic quantization.

Test Plan:

Reviewers:

Subscribers:

Tasks:

Tags:

Differential Revision: [D30658654](https://our.internmc.facebook.com/intern/diff/D30658654)

[ghstack-poisoned]
